tests/baselines/reference/privateNameErrorsOnNotUseDefineForClassFieldsInEsNext(target=es2020).js

```diff
@@ -0,0 +1,67 @@
+//// [privateNameErrorsOnNotUseDefineForClassFieldsInEsNext.ts]
+class Test {
+    #prop = 0 
+    static dd = new Test().#prop; // Err
+    static ["X_ z_ zz"] = class Inner {
+        #foo  = 10   
+        m() {
+            new Test().#prop // Err
+        }
+        static C = class InnerInner {
+            m() {
+                new Test().#prop // Err
+                new Inner().#foo; // Err
+            }
+        }
+
+        static M(){
+            return class {
+                m() {
+                    new Test().#prop // Err
+                    new Inner().#foo; // OK
+                }
+            }
+        } 
+    }
+}

```

Comment:
Would be good to add some private methods and accessor cases to this test either here on in 42458, depending on which one goes in first.

Reply:
I expect this to go in first. (FYI we will have conflict with private methods and accessors since this adds some extra messages)